resource/subfunctions/appc_session.md

```diff
@@ -0,0 +1,283 @@
+
+&nbsp;
+
+Alter an existing **APPC Session** profile in the **APPCLU** class.
+
+#### 📥 Parameters
+* `net_id`<br>
+  The **Netword ID** of the **APPC Session** being altered.
+
+* `local_lu`<br>
+  The name of the **Local Logical Unit** of the **APPC Session** being altered.
+
+* `partner_lu`<br>
+  The name of the **Partner Logical Unit** of the **APPC Session** being altered.
+
+* `traits`<br>
+  A dictionary of **traits/attributes** that should be given to the resource. See [Traits](../../base/segments_traits_operators#traits) to see what all of the valid **Resource Traits** are.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Security Result dictionary** or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AlterOperationError`<br>
+  Raises `AlterOperationError` when the **z/OS userid** supplied cannot be altered because it does **NOT** exist.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** provided contains one or more **unknown** traits.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.alter_appc_session("TSTNET", "TSTLOCLU", "TSTPRTLU", traits={"session:locked": True})
+{'securityResult': {'resource': {'name': 'TSTNET.TSTLOCLU.TSTPRTLU', 'class': 'APPCLU', 'operation': 'set', 'requestId': 'ResourceRequest', 'info': ['Definition exists. Add command skipped due  to precheck option'], 'commands': [{'safReturnCode': 0, 'returnCode': 0, 'reasonCode': 0, 'image': 'RALTER  APPCLU             (TSTNET.TSTLOCLU.TSTPRTLU)  SESSION  (   LOCK        )'}]}, 'returnCode': 0, 'reasonCode': 0}}
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"TSTNET.TSTLOCLU.TSTPRTLU",
+      "class":"APPCLU",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "info":[
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RALTER  APPCLU             (TSTNET.TSTLOCLU.TSTPRTLU)  SESSION  (   LOCK        )"
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.extract_appc_session()`
+
+```python
+def extract_appc_session(
+    self, net_id: str, local_lu: str, partner_lu: str
+) -> Union[dict, bytes]:
+```
+
+#### 📄 Description
+
+&nbsp;
+
+{: .experimental }
+> _So far, no tested **Segments** and **Traits** are considered **Stable** for profile extract of the **session** segment. This is how the function will work when **Traits** are made **Stable** for extract._

```

Comment:
Change `for profile extract of the` to `when extracting the`

Reply:
Also change `when **Traits** are made **Stable** for extract.` to `when **Traits** for the `session` segment are made **Stable**.`

Reply:
Changes **session** to `session`.

Reply:
all fixed

Reply:
`the` is repeated twice.